backend/test/rules/new_game_test.clj

```diff
@@ -25,10 +26,25 @@
 
   ; Rows begin empty
   (expect
-    [[] [] [] [] []]
+    (repeat 5 [])
```

Comment:
I prefere to see what I will get in a test, but ok...

Reply:
This I can see both ways, I changed one direction and back like 6-7 times 😂
I would just ask that we be consistent throughout